src/main/kotlin/uk/gov/dluhc/printapi/service/tempcard/ExplainerPdfService.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dluhc.printapi.service.tempcard
+
+import org.springframework.stereotype.Service
+import uk.gov.dluhc.printapi.client.ElectoralRegistrationOfficeManagementApiClient
+
+@Service
+class ExplainerPdfService(
+    private val eroClient: ElectoralRegistrationOfficeManagementApiClient,
+    private val explainerPdfFactory: ExplainerPdfFactory
+) {
+
+    fun generateExplainerPdf(gssCode: String): PdfFile {
+        val eroDto = eroClient.getEro(gssCode)
+        val contents = explainerPdfFactory.createPdfContents(eroDto, gssCode)
+        return PdfFile("temporary-certificate-explainer-document-$gssCode.pdf", contents)
+    }
+}
+
+class PdfFile(
```

Comment:
Minor - is it worth being a class/source file in its own right? I'll be wanting the same class when I come to populate the temp cert template?
Also, could it be a data class?

Reply:
Done